test/cmdlineTests.sh

```diff
@@ -133,13 +135,8 @@ function test_solc_behaviour()
 
     if [[ "$exit_code_expected" = "" ]]; then exit_code_expected="0"; fi
 
-    local solc_command="$SOLC ${filename} ${solc_args}"
```

Comment:
Actually this variable is also referenced at https://github.com/ethereum/solidity/blob/94053c9a526cc55f79b65a01d05e3c1778fa5312/test/cmdlineTests.sh#L173

Reply:
Ah thanks! Re-added the variable.